src/sentry/migrations/0619_monitors_migrate_is_muted.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 3.2.23 on 2023-12-08 20:47
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def migrate_is_muted(apps, schema_editor):
+    Monitor = apps.get_model("sentry", "Monitor")
+    query = Monitor.objects.filter(status=1)
+
+    # Query muted monitors
+    for monitor in RangeQuerySetWrapperWithProgressBar(query):
+        monitor.is_muted = True
+        monitor.save()
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
```

Comment:
How many rows will you be updating? If it is more than 10k you should consider a is_dangerous/postdeploy migration as you're not likely to be able to complete all the row mutations in the 5s timeout that we have for migrations.

Reply:
We're good. There are currently less than 3k "mute" status monitors.